packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -1835,3 +1854,84 @@ function normalizeResolversNonOptionalTypename(
     ...input,
   };
 }
+
+function checkIfObjectTypeHasAbstractTypesRecursively(
+  baseType: GraphQLObjectType,
+  result: {
+    isObjectWithAbstractType: boolean;
+    checkedTypesWithNestedAbstractTypes: Record<string, { hasNestedAbstractTypes: 'yes' | 'no' | 'checking' }>;
```

Comment:
can it have a better name ? for me, anything which starts with `has*` should be a boolean

Reply:
True! It started as `boolean` but then I saw the need to change to 3 different options, but didn't review the name.

Do you have suggestions? I'm thinking a few options but I'm unsure about them:
- `value`
- `result`
- `checkStatus`

Reply:
checkStatus For me is good